src/main/java/at/hannibal2/skyhanni/features/rift/RiftAPI.kt

```diff
@@ -21,16 +21,24 @@ object RiftAPI {
 
     val farmingTool by lazy { "FARMING_WAND".asInternalName() }
 
+    private val blowgun by lazy  { "BERBERIS_BLOWGUN".asInternalName() }
+
+    val ItemStack.isBlowgun get() = getInternalName() == blowgun
+
     fun ItemStack.motesNpcPrice(): Double? {
         val baseMotes = motesPrice[getInternalName()] ?: return null
         val burgerStacks = config.motes.burgerStacks
         val pricePer = baseMotes + (burgerStacks * 5) * baseMotes / 100
         return pricePer * stackSize
     }
 
+    var inRiftRace = false
+    var showButtons = false
+    var allButtonsHit = false
+
     fun inLivingCave() = LorenzUtils.skyBlockArea == "Living Cave"
     fun inLivingStillness() = LorenzUtils.skyBlockArea == "Living Stillness"
     fun inStillgoreChateau() = LorenzUtils.skyBlockArea.let { it == "Stillgore Château" || it == "Oubliette" }
-
     fun inDreadfarm() = LorenzUtils.skyBlockArea == "Dreadfarm"
+    fun inWestVillage() = LorenzUtils.skyBlockArea.let { it == "West Village" || it == "Infested House" }
```

Comment:
You might want to use the area position from the island graph feature for this check to.be more precise.

Reply:
so:
`fun inWestVillage() =IslandAreas.currentAreaName == "West Village" || IslandAreas.currentAreaName == "Infested House"?`